### PR TITLE
Add "jump to end" control to simulation player

### LINKED ISSUE #31: "Jump to end" control button for playback
It would be nice if there was a `>|` button that jumped to the end of the playback.

### PR DESCRIPTION
**Pull request by jnowaczek**
Closes #31 

I feel sort of bad fixing this, but in my defense the issue is 2 years old...

image

Edit: add purty picture

### CODE REVIEW

File: CYLGame/www/static/player.js [hunk 1]
```diff
@@ -92,6 +92,11 @@ class Player {
         .click(() => {
           this.on_faster_click();
         }));
+      this.btn_bar.append($("<button>")
+        .text(">|")
```
Comment (ChandlerSwift) [inline CYLGame/www/static/player.js:96]: Rather than approximating these symbols, do we want to just use the unicode? (perhaps ⏭️ for this one)

Comment (jnowaczek) [reply CYLGame/www/static/player.js:96]: IMHO the whole player could use a little UX tlc, but at least for now it matches the others

Comment (ChandlerSwift) [reply CYLGame/www/static/player.js:96]: Works for me! I just noticed that I _do_ in fact need to approve for you to merge, so consider this my blessing

**Review by ChandlerSwift — APPROVED** [review]

File: CYLGame/www/static/player.js (hunk 1 above)
Comment (ChandlerSwift) [reply CYLGame/www/static/player.js:96]: I just want to make sure you notice that the vertical bar character isn't vertically centered with respect to the greater than symbol. https://xkcd.com/1015/ ;)

Comment (jnowaczek) [reply CYLGame/www/static/player.js:96]: Perhaps it should be https://decodeunicode.org/en/u+23d0 instead 🤔

Comment (jnowaczek) [reply CYLGame/www/static/player.js:96]: image
It actually looks better 😮

**Pull request merged**